quantum/via_ensure_keycode.h

```diff
@@ -0,0 +1,271 @@
+_Static_assert(RGB_VAD == 23754, "");
+_Static_assert(RGB_SPI == 23755, "");
+_Static_assert(RGB_SPD == 23756, "");
+_Static_assert(RGB_M_P == 23757, "");
+_Static_assert(RGB_M_B == 23758, "");
+_Static_assert(RGB_M_R == 23759, "");
+_Static_assert(RGB_M_SW == 23760, "");
+_Static_assert(RGB_M_SN == 23761, "");
+_Static_assert(RGB_M_K == 23762, "");
+_Static_assert(RGB_M_X == 23763, "");
+_Static_assert(RGB_M_G == 23764, "");
+_Static_assert(RGB_M_T == 23765, "");
+
+_Static_assert(KC_F13 == 104, "");
+_Static_assert(KC_F14 == 105, "");
+_Static_assert(KC_F15 == 106, "");
+_Static_assert(KC_F16 == 107, "");
+_Static_assert(KC_F17 == 108, "");
+_Static_assert(KC_F18 == 109, "");
+_Static_assert(KC_F19 == 110, "");
+_Static_assert(KC_F20 == 111, "");
+_Static_assert(KC_F21 == 112, "");
+_Static_assert(KC_F22 == 113, "");
+_Static_assert(KC_F23 == 114, "");
+_Static_assert(KC_F24 == 115, "");
+_Static_assert(KC_PWR == 165, "");
+_Static_assert(KC_SLEP == 166, "");
+_Static_assert(KC_WAKE == 167, "");
+_Static_assert(KC_EXEC == 116, "");
+_Static_assert(KC_HELP == 117, "");
+_Static_assert(KC_SLCT == 119, "");
+_Static_assert(KC_STOP == 120, "");
+_Static_assert(KC_AGIN == 121, "");
+_Static_assert(KC_UNDO == 122, "");
+_Static_assert(KC_CUT == 123, "");
+_Static_assert(KC_COPY == 124, "");
+_Static_assert(KC_PSTE == 125, "");
+_Static_assert(KC_FIND == 126, "");
+_Static_assert(KC_CALC == 178, "");
+_Static_assert(KC_MAIL == 177, "");
+_Static_assert(KC_MSEL == 175, "");
+_Static_assert(KC_MYCM == 179, "");
+_Static_assert(KC_WSCH == 180, "");
+_Static_assert(KC_WHOM == 181, "");
+_Static_assert(KC_WBAK == 182, "");
+_Static_assert(KC_WFWD == 183, "");
+_Static_assert(KC_WSTP == 184, "");
+_Static_assert(KC_WREF == 185, "");
+_Static_assert(KC_WFAV == 186, "");
+_Static_assert(KC_BRIU == 189, "");
+_Static_assert(KC_BRID == 190, "");
+_Static_assert(KC_MPRV == 172, "");
+_Static_assert(KC_MNXT == 171, "");
+_Static_assert(KC_MUTE == 168, "");
+_Static_assert(KC_VOLD == 170, "");
+_Static_assert(KC_VOLU == 169, "");
+_Static_assert(KC_MSTP == 173, "");
+_Static_assert(KC_MPLY == 174, "");
+_Static_assert(KC_MRWD == 188, "");
+_Static_assert(KC_MFFD == 187, "");
+_Static_assert(KC_EJCT == 176, "");
+_Static_assert(KC_MS_U == 240, "");
+_Static_assert(KC_MS_D == 241, "");
+_Static_assert(KC_MS_L == 242, "");
+_Static_assert(KC_MS_R == 243, "");
+_Static_assert(KC_BTN1 == 244, "");
+_Static_assert(KC_BTN2 == 245, "");
+_Static_assert(KC_BTN3 == 246, "");
+_Static_assert(KC_BTN4 == 247, "");
+_Static_assert(KC_BTN5 == 248, "");
+_Static_assert(KC_WH_U == 249, "");
+_Static_assert(KC_WH_D == 250, "");
+_Static_assert(KC_WH_L == 251, "");
+_Static_assert(KC_WH_R == 252, "");
+_Static_assert(KC_ACL0 == 253, "");
+_Static_assert(KC_ACL1 == 254, "");
+_Static_assert(KC_ACL2 == 255, "");
+_Static_assert(KC_LCAP == 130, "");
+_Static_assert(KC_LNUM == 131, "");
+_Static_assert(KC_LSCR == 132, "");
+
+_Static_assert(MACRO00 == 0x5F12, "");
```

Comment:
what about USER00-USER15?  As well as FN_MO13, and FN_MO23? 

```suggestion
_Static_assert(FN_MO13 == 0x5F10, "");
_Static_assert(FN_MO23 == 0x5F11, "");
_Static_assert(MACRO00 == 0x5F12, "");
```

Reply:
these are up top but i'll move them down